extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -27,13 +36,32 @@ function register_block() {
 }
 add_action( 'init', __NAMESPACE__ . '\register_block' );
 
+/**
+ * Get current URL.
+ *
+ * @return string Current URL.
+ */
+function get_current_url() {
+	if ( isset( $_SERVER['HTTP_HOST'] ) ) {
+		$host = wp_unslash( $_SERVER['HTTP_HOST'] );
```

Comment:
This seems to be triggering a PHPCS error:

```
 Error  (PHPCS-CHANGED WordPress.Security.ValidatedSanitizedInput.InputNotSanitized) Detected usage of a non-sanitized input variable: $_SERVER['HTTP_HOST']
    Detected usage of a non-sanitized input variable: $_SERVER['HTTP_HOST']

              43  */
              44 function get_current_url() {
              45 	if ( isset( $_SERVER['HTTP_HOST'] ) ) {
    >>>       46 		$host = wp_unslash( $_SERVER['HTTP_HOST'] );
```

Same below:

```
  Error  (PHPCS-CHANGED WordPress.Security.ValidatedSanitizedInput.InputNotSanitized) Detected usage of a non-sanitized input variable: $_SERVER['REQUEST_URI']
    Detected usage of a non-sanitized input variable: $_SERVER['REQUEST_URI']

              48 		$host = wp_parse_url( home_url(), PHP_URL_HOST );
              49 	}
              50 	if ( isset( $_SERVER['REQUEST_URI'] ) ) {
    >>>       51 		$path = wp_unslash( $_SERVER['REQUEST_URI'] );
```

Reply:
Shall I just `// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized` these?

Reply:
Sanitization is done via `esc_url_raw()` below.

Reply:
```suggestion
		$host = wp_unslash( $_SERVER['HTTP_HOST'] ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```